core/shape-parts.cpp

```diff
@@ -39,6 +39,32 @@ const Path& BasicBezier::generate(Point endPoint, Point transitionPoint) {
   return m_path;
 }
 
+const Path& FlatBezier::generate(Point endPoint, Point transitionPoint) {
+  MCFixed flatEndY = m_flatWidth / -2; // WHY DO I HAVE TO MAKE THIS NEGATIVE TO GET IT TO WORK???
+  if (endPoint.X < transitionPoint.X) {
+    Path ep;
+    ep.push_back(endPoint);
```

Comment:
Let's assume a `m_flatWidth` of `4`. `endPoint` is (0,0). So starting here is good.